source/adressering-av-undertegner.rst

```diff
@@ -7,6 +7,8 @@ You may address signers for :ref:`portal flow <signing-in-portal-flow>` in two d
 1. By email and/or SMS (only available for private organizations)
 2. By national identity number
 
+The signed document will only include signatures from these signers, so remember to include signers from the sender side as well, if necessary.
```

Comment:
Litt usikker på setningen her, så åpen for forslag! Tanken er å påpeke at vi ikke legger inn noe informasjon i det signerte dokumentet om avsender hvis de ikke har gjort det selv.

Reply:
Kanskje noe sånt, siden man snakker om adressering av undertegnere?

```suggestion
The signed document will only include signatures from the *addressed* signers, so remember to include signers from the sender side as well, if applicable.
```

Bra presisering 👍